tensorflow_recommenders_addons/dynamic_embedding/core/kernels/redis_impl/md5.cc

```diff
@@ -0,0 +1,305 @@
+/* MD5.C - RSA Data Security, Inc., MD5 message-digest algorithm
+ */
+
+/* Copyright (C) 1991-2, RSA Data Security, Inc. Created 1991. All
```

Comment:
Can we use this file in the repo for open source target?

Reply:
copy from https://datatracker.ietf.org/doc/html/rfc1321